src/coprocessor/codec/batch/lazy_column.rs

```diff
@@ -370,89 +282,14 @@ mod benches {
         b.iter(|| {
             let column = test::black_box(&mut column);
             for _ in 0..1000 {
-                column.push_raw(test::black_box(&val))
+                column.mut_raw().push(test::black_box(&val))
             }
             test::black_box(&column);
             column.clear();
             test::black_box(&column);
         });
     }
 
-    #[bench]
-    fn bench_lazy_batch_column_push_raw_9bytes(b: &mut test::Bencher) {
-        let mut column = LazyBatchColumn::raw_with_capacity(1000);
-        let val = vec![0; 9];
-        b.iter(|| {
-            let column = test::black_box(&mut column);
-            for _ in 0..1000 {
-                column.push_raw(test::black_box(&val))
-            }
-            test::black_box(&column);
-            column.clear();
-            test::black_box(&column);
-        });
-    }
-
-    /// 10 bytes > inline size for LazyBatchColumn, which will be slower.
-    /// This benchmark shows how slow it will be.
-    #[bench]
-    fn bench_lazy_batch_column_push_raw_10bytes(b: &mut test::Bencher) {
```

Comment:
Could you post benchmark result of these benches before and after the refinement to the description of the PR?

Reply:
In our internal benchmark, a case has a performance regression at about "decrease 10%" before this PR. After this PR is merged, it will be changed to "improve 30%".